### PR TITLE
chore: fix issues with docker compose update GitHub workflow

### PR DESCRIPTION
**Pull request by edgarvonk**
Fixed issues with Docker Compose update GitHub workflow. Made all Docker image URLs absolute so that https://github.com/sbe-arg/simple-compose-service-updates can update them. Ignored https://github.com/eugenmayer/officeconverter because that has 'vX.X.X' versions which are deemed newer while they are not.

Solves PZ-4193

### CODE REVIEW

File: .github/workflows/update-docker-compose.yml [hunk 1]
```diff
@@ -18,14 +18,24 @@ permissions:
...
       - uses: actions/checkout@v4
 
+      - name: setup-git
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"  
+
       - name: simple-compose-service-updates
         uses: sbe-arg/simple-compose-service-updates@v1.1.0
+        with:
```
Comment (edgarvonk) [inline .github/workflows/update-docker-compose.yml:36]: We can also configure the thing to not create PRs at all but instead just run. Then I guess you need to check the run log to these if there are updates. Maybe better.. What do you guys think?

Comment (bas-info-nl) [reply .github/workflows/update-docker-compose.yml:36]: I'd prefer to see PRs

File: docker-compose.yaml [hunk 2]
```diff
@@ -17,6 +17,7 @@ services:
       - DB_VENDOR=postgres
       - DB_ADDR=keycloak-database
       - DB_USER=keycloak
+      - DB_USER=keycloak
```
Comment (bas-info-nl) [inline docker-compose.yaml:20]: doubled up ?

**Review by bas-info-nl — APPROVED** [review]

File: docker-compose.yaml (hunk 2 above)
Comment (edgarvonk) [reply docker-compose.yaml:20]: Huh? No idea where this came from.. Will fix..

**Pull request merged**